overlord/overlord.go

```diff
@@ -102,6 +103,7 @@ func New() (*Overlord, error) {
 	o.hookMgr = hookMgr
 	o.stateEng.AddManager(o.hookMgr)
 
+	hooks.SetupHooks(o.hookMgr)
```

Comment:
I don't understand the point. hookstate.Manager is _within_ hookstate. It wouldn't have to import anything.